src/main/java/net/mcreator/element/converter/fv13/ProcedureSpawnGemAddPickupDelayFixer.java

```diff
@@ -73,10 +74,10 @@ protected String fixXML(String xml) throws Exception {
 			String type = element.getAttribute("type");
 			if (type != null && strings_tofix_types.contains(type)) {
 				Element value = doc.createElement("value");
-				value.setAttribute("name", "string");
-				Element deps_block = doc.createElement("block");
-				deps_block.setAttribute("type", "string");
-				value.appendChild(deps_block);
+				value.setAttribute("name", "pickUpDelay");
```

Comment:
I did not test this code yet, but I can't see the location here where you actually set the value of said math block.

Reply:
I did test the code and it works. The value is `0` by default

Reply:
I did not say it will not work, just said default value is not the one that should be.

Reply:
As I explained before I found how to set a default value, nut I did not found the way to set this default value in the UI.

Reply:
Try a bit more, otherwise I can try next week, my weekend will be a bit busy ;)